build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -21,11 +21,22 @@ const jsModuleErrorsExtension = '.ddc.js.errors';
 const jsModuleExtension = '.ddc.js';
 const jsSourceMapExtension = '.ddc.js.map';
 const metadataExtension = '.ddc.js.metadata';
+const fullKernelExtension = '.ddc.full.dill';
 
 /// A builder which can output ddc modules!
 class DevCompilerBuilder implements Builder {
   final bool useIncrementalCompiler;
 
+  /// Make ddc generate full dill for libraries it compiles.
+  ///
+  /// Full dill is only generated in debug mode and does not include any
+  /// dependent libraries or SDK dill.
+  ///
+  /// Full dill is used by the expression compilation service run by the
```

Comment:
I think somewhere in this comment we should mention that it implies an extra file. As currently phrased it feels to me like different content in the same file.

Reply:
will do